hedera-node/src/main/java/com/hedera/services/fees/calculation/AwareFcfsUsagePrices.java

```diff
@@ -118,10 +118,10 @@ public FeeData pricesGiven(HederaFunctionality function, Timestamp at) {
 			return usagePrices;
 		} catch (Exception e) {
 			log.warn(
-					"Only default usage prices available for function {} @ {}! ({})",
+					"Default usage price will be used, no specific usage prices available for function {} @ {}!",
 					function,
-					Instant.ofEpochSecond(at.getSeconds(), at.getNanos()),
-					e);
+					Instant.ofEpochSecond(at.getSeconds(), at.getNanos())
+					);
```

Comment:
Please add a test for this change. We have `com.hedera.test.mocks.MockAppender` to facilitate this test of loggings.